include/constants/battle_config.h

```diff
@@ -130,6 +130,10 @@
 #define B_PARALYZE_ELECTRIC         GEN_6 // In Gen6+, Electric type Pokémon can't be paralyzed.
 #define B_POWDER_GRASS              GEN_6 // In Gen6+, Grass type Pokémon are immune to powder and spore moves.
 
+// Critical Capture
+#define B_CRITICAL_CAPTURE            TRUE // if set to TRUE, critical capture will be enabled
+#define B_CATCHING_CHARM_BOOST        20   // % boost in critical capture odds if player has the catching charm
```

Comment:
Various things:
- Please put this section between Interface settings and Other and align according to them.
- Use proper capitalization and punctuation in the comments. (dots at the end of phrases)
- I think features that are from other generations should use the `GEN_X` format, while stuff specific to the engine should use `TRUE/FALSE`. Please change `B_CRITICAL_CAPTRUE` to use the gen format.
- Because of that, could you also change `B_ABILITY_POP_UP` to use the gen format?
- A good setting to add would be the chance of Critical capture itself, to justify it having its own section.

Reply:
Talked about it on the Discord and decided that only changes 1, 2 and 5 should apply.

Reply:
did 1 and 2, and made critical capture odds based on a ratio of NUM_SPECIES and the gen5 species count, eg. for gen3 it would be `386 * 30/650` for the first multiplier tier.